extensions/pyRevitCore.extension/pyRevit.tab/pyRevit.panel/Settings.smartbutton/script.py

```diff
@@ -389,42 +388,27 @@ def _setup_addinfiles(self):
             {str(x.Product.ProductYear):x
              for x in PyRevit.PyRevitAttachments.GetAttachments()}
 
-        for rvt_ver, checkbox in self._addinfiles_cboxes.items():
-            if rvt_ver in attachments:
-                if rvt_ver != HOST_APP.version:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            attachments[rvt_ver].Product,
-                            ''
-                            )
+        for rvt_cb in self.supported_revit_versions_CB:
+            if rvt_cb.Version in attachments:
+                if rvt_cb.Version != HOST_APP.version:
+                    rvt_cb.Content = rvt_cb.Content + self._make_product_name(attachments[rvt_cb.Version].Product,'')
                 else:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            attachments[rvt_ver].Product,
-                            self.get_locale_string("RevitAttachment.Current")
-                            )
-
-                checkbox.IsChecked = True
-                if attachments[rvt_ver].AttachmentType == \
-                        PyRevit.PyRevitAttachmentType.AllUsers:
-                    checkbox.IsEnabled = False
-                    checkbox.Content += self.get_locale_string("RevitAttachment.AllUsers")
+                    rvt_cb.Content = rvt_cb.Content + self._make_product_name(attachments[rvt_cb.Version].Product,self.get_locale_string("RevitAttachment.Current"))
+                rvt_cb.IsChecked = True
+                if attachments[rvt_cb.Version].AttachmentType == PyRevit.PyRevitAttachmentType.AllUsers:
+                    rvt_cb.IsEnabled = False
+                    rvt_cb.Content += self.get_locale_string("RevitAttachment.AllUsers")
                 else:
-                    checkbox.IsEnabled = True
+                    rvt_cb.IsEnabled = True
             else:
-                if rvt_ver in installed_revits:
-                    checkbox.Content = \
-                        self._make_product_name(
-                            installed_revits[rvt_ver],
-                            self.get_locale_string("RevitAttachment.NotAttached")
-                            )
-                    checkbox.IsEnabled = True
-                    checkbox.IsChecked = False
+                if rvt_cb.Version in installed_revits:
+                    rvt_cb.Content = rvt_cb.Content + self._make_product_name(installed_revits[rvt_cb.Version],self.get_locale_string("RevitAttachment.NotAttached"))
```

Comment:
use += and avoid too long lines
```suggestion
                    rvt_cb.Content += self._make_product_name(
                        installed_revits[rvt_cb.Version],
                        self.get_locale_string("RevitAttachment.NotAttached")
                    )
```

Reply:
i did that